api/app/db/crud/hfi_calc.py

```diff
@@ -175,6 +178,24 @@ def get_stations_for_affected_planning_areas(session: Session, request: HFIAdmin
         .all()
 
 
+def unready_planning_areas(session: Session,
+                           date_range: Optional[DateRange],
+                           fire_centre_id: int,
+                           username: str,
+                           planning_area_ids):
+    if date_range is None:
+        return
+    now = get_utc_now()
+    query = update(HFIReady).values(
+        {HFIReady.ready: False, HFIReady.update_user: username, HFIReady.update_timestamp: now})\
+        .where(HFIReady.planning_area_id.in_(planning_area_ids))\
+        .where(HFIReady.hfi_request_id == HFIRequest.id)\
+        .where(HFIRequest.fire_centre_id == fire_centre_id)\
+        .where(HFIRequest.prep_start_day >= date_range.start_date).\
```

Comment:
I think we need to add some explicit comments around **why** this is happening. Looking at the code it isn't clear why we're only changing requests after the "start_date", as opposed to all requests. I would guess it's business logic, there's some rule that states that we don't "unready" old requests. But what's to stop me from changing my prep period to 1997, and then editing the stations?
Why discriminate? Why not just mark them all as unready - it should be a fairly fast database operation?

Reply:
Sure. I guess the behaviour we need to accept is that all "readied" HFI prep periods in the past are now unready. The model in my head is that a station modification affects current a future prep periods, hence the `gte` start date discrimination.

Reply:
I would go with whatever Tess thinks is more appropriate on this.

Reply:
I can't test this in prod, so this is the logic I'd advocate for: 
A - if the station changes are applied historically, un-ready all history as well - be consistent (I think this is the reality?)
or
B - if the station changes are only applied today onwards, and historical station lists are saved, then only un-ready today forward
- _If there were a stronger use case for recording history in the product this would have made sense, but because of the actuals context, we rely on PDF records instead as an MVP_

Reply:
Ok sounds good, I'll make the changes to unready all states for a planning area for added/removed stations.